include/lmms_math.h

```diff
@@ -360,6 +360,22 @@ static inline int numDigitsAsInt(float f)
 	return digits;
 }
 
+// Helper functions to 
+static inline float minEps()
+{
+	return 1.0e-10f;
+}
+
+static inline bool FloatIsEqual(float x, float y)
```

Comment:
I  thought of using a == operator instead of this function but gpt suggested it might lead to accuracy issues.